### PR TITLE
Support for markup type

### LINKED ISSUE #743: Can create unnamed organization or person
I cannot create unnamed organization or person on first try, but when I edit it's name, I can leave it empty with no error. Also placeholder for organizations seem to be "Software Engineer"

### PR DESCRIPTION
**Pull request by haiodo**
1. TypeMarkdown
2. StyledTextBox - for by click editing of big descriptions.
3. Svele + Hot Module Reload.
4. Closes #743

Screenshot 2022-02-05 at 00 02 00

Screenshot 2022-02-05 at 00 02 07

Screenshot 2022-02-05 at 00 02 53

Screenshot 2022-02-05 at 00 02 18

Screenshot 2022-02-05 at 00 02 36 Signed-off-by: Andrey Sobolev

### CODE REVIEW

File: plugins/recruit-resources/src/components/EditVacancy.svelte [hunk 1]
```diff
@@ -81,13 +83,21 @@
       <div class="flex-col box">
         {#if selected === 0}
           <Grid column={1} rowGap={1.5}>
-            <EditBox label={recruit.string.VacancyName} bind:value={object.name} placeholder="Software Engineer" maxWidth="39rem" focus on:change={() => {onChange('name', object.name)}}/>
-            <EditBox label={recruit.string.Description} bind:value={object.description} placeholder='Description' maxWidth="39rem" focus on:change={() => {onChange('description', object.description)}}/>
+            <EditBox label={recruit.string.VacancyName} bind:value={object.name} placeholder={recruit.string.VacancyPlaceholder} maxWidth="39rem" focus on:change={() => {
+              if (object.name.trim().length > 0) {
+                  onChange('name', object.name)
+              } else {
+                query.query(recruit.class.Vacancy, { _id }, result => {
```
Comment (BykhovDenis) [inline plugins/recruit-resources/src/components/EditVacancy.svelte]: Why we need it here?

File: plugins/task-resources/src/components/EditIssue.svelte [hunk 2]
```diff
@@ -27,6 +28,7 @@
   const client = getClient()
 
   function change (field: string, value: any) {
+    console.log('change', value)
```
Comment (BykhovDenis) [inline plugins/task-resources/src/components/EditIssue.svelte]: extra log

File: plugins/recruit-resources/src/components/EditVacancy.svelte [hunk 3]
```diff
@@ -33,14 +33,16 @@
   const dispatch = createEventDispatcher()
 
   const client = getClient()
+  let oldName: string = ''
```
Comment (BykhovDenis) [inline plugins/recruit-resources/src/components/EditVacancy.svelte]: Why we need it here? ( I think it or query refresh is extra)

File: packages/text-editor/lang/en.json [hunk 4]
```diff
@@ -1,6 +1,10 @@
 {
   "string": {    
     "Suggested": "SUGGESTED",
-    "NoItems": "No items"
+    "NoItems": "No items",
+    "EditorPlaceholder": "Start typing2...",
```
Comment (BykhovDenis) [inline packages/text-editor/lang/en.json]: '2'? typo?

**Review by BykhovDenis — APPROVED** [review]

File: plugins/recruit-resources/src/components/EditVacancy.svelte (hunk 3 above)
Comment (haiodo) [reply plugins/recruit-resources/src/components/EditVacancy.svelte]: Fixed.

File: plugins/recruit-resources/src/components/EditVacancy.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/recruit-resources/src/components/EditVacancy.svelte]: updated to call updateObject

**Pull request merged**